rapids/rapids.sh

```diff
@@ -12,12 +12,15 @@ readonly SPARK_VERSION_ENV=$(spark-submit --version 2>&1 | sed -n 's/.*version[[
 
 if [[ "${SPARK_VERSION_ENV}" == "3"* ]]; then
   readonly DEFAULT_CUDF_VERSION="0.17"
-  readonly DEFAULT_SPARK_RAPIDS_VERSION="0.2.0"
+  readonly DEFAULT_SPARK_RAPIDS_VERSION="0.3.0"
+  readonly DEFAULT_XGBOOST_VERSION="1.3.0"
+  readonly DEFAULT_XGBOOST_GPU_SUB_VERSION="0.1.0"
   # TODO: uncomment when Spark 3.1 jars will be released.
   # readonly SPARK_VERSION="${SPARK_VERSION_ENV}"
   readonly SPARK_VERSION="3.0"
 else
   readonly DEFAULT_CUDF_VERSION="0.9.2"
+  readonly DEFAULT_XGBOOST_VERSION="1.0.0"
   readonly DEFAULT_SPARK_RAPIDS_VERSION="Beta5"
```

Comment:
Should we change this line DEFAULT_SPARK_RAPIDS_VERSION to DEFAULT_XGBOOST_GPU_SUB_VERSION also for spark 2.x so that line 135 can get the correct xgboost version?
Of course after you change it, you should also change line 106 & 109.

Reply:
could you check the latest commit and see if that looks right for you